spec/movements_spec.rb

```diff
@@ -22,5 +22,28 @@
     expect(movements.change_orientation_anticlockwise("E")).to eq("N")
   end
 
+  it "returns 2 as value for coordinate x if robot starts from [1,1] and moves East" do
+    coordinates = [1,1]
```

Comment:
Very specific test descriptions.  I'd try to aim for more generality e.g. it 'moves robot east' is enough.